src/components/sync-sites-modal-selector.tsx

```diff
@@ -129,7 +130,18 @@ function SearchSites( {
 				__nextHasNoMarginBottom={ true }
 			/>
 			<p className="a8c-helper-text text-gray-500">
-				{ __( 'Syncing is supported for WP.com sites on the Business plan or above.' ) }
+				{ __( "Can't find your site? " ) }
+				<Button
+					variant="link"
+					onClick={ () =>
+						getIpcApi().openURL(
+							'https://developer.wordpress.com/docs/developer-tools/studio/sync/'
```

Comment:
Can we store a URL in a constant and use our URL localization function?

Reply:
That's a good catch! Thanks! I have addressed it in https://github.com/Automattic/studio/pull/1406/commits/a1857dd423d3d3419e2fb03a6aa87e1bbc2ebb22.